telemetry-aware-scheduling/pkg/cache/autoupdating.go

```diff
@@ -81,7 +82,7 @@ func (n *AutoUpdatingCache) ReadMetric(metricName string) (metrics.NodeMetricsIn
 			return metric, nil
 		}
 	}
-	return metrics.NodeMetricsInfo{}, errors.New("no metric " + metricName + " found")
+	return metrics.NodeMetricsInfo{}, errors.New("no metric " + metricName + " found yet")
```

Comment:
Remove this 'yet' the context isn't fully clear

Reply:
change made.